geomstats/geometry/discrete_curves.py

```diff
@@ -842,3 +842,340 @@ def g_criterion(srv, srv_norms):
             nb_iter += 1
 
         return proj
+
+
+class ElasticMetric(RiemannianMetric):
+    """Elastic metric defined using the F_transform.
+
+    See [NK2018]_ for details.
+
+    Parameters
+    ----------
+    a: float
+        Bending parameter.
+    b: float
+        Stretching parameter.
+
+    References
+    ----------
+    .. [KN2018] S. Kurtek and T. Needham,
+    "Simplifying transforms for general elastic metrics on the space of
+    plane curves", arXiv:1803.10894 [math.DG], 29 Mar 2018.
+    """
+
+    def __init__(self, a, b):
```

Comment:
Should we call the SRVMetric if a = 1 and b = 1? Or at least explain in the docstring how the two metrics relate? @Florent-Michel

Reply:
`SRVMetric` and `ElasticMetric(1, 0.5)` are indeed almost the same (up to a factor which depends on the number of sampling points) because I made different normalization choices

Reply:
I think that for coherence it is important that the results of ```SRVMetric``` and ```ElasticMetric(1, 0.5)``` are the same. If it's just a normalization choice, can it be easily changed in ```SRVMetric``` ?